src/components/ComingSoon.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="coming-soon_wrapper">
+    <div class="headings">
+      <h3>SOON IN THE CINEMA</h3>
+      <h3>SOON</h3>
+      <h3>SEE ALL</h3>
+    </div>
+    <div class="soon-movies_wrapper">
+      <MovieCard
+        title="Predator"
+        img="/src/assets/img/predator.png"
+        description="A team of commandos on a mission in a Central American jungle find themselves haunted by an extraterrestrial warrior."
+      ></MovieCard>
```

Comment:
very good idea to make it a component 👍 

I suggest to also move movie data to `data` as an array and make it a `v-for` loop  - that would be perfect!